caiman/source_extraction/cnmf/initialization.py

```diff
@@ -283,11 +278,6 @@ def initialize_components(Y, K=30, gSig=[5, 5], gSiz=None, ssub=1, tsub=1, nIter
 
     """
     method = method_init
-    if method == 'local_nmf':
```

Comment:
Note that I'm removing this because it never worked (in fact could not have worked because below it depends on imports that didn't work)